### DISCUSSION TITLE
PHP mailer native mail

### DISCUSSION THREAD

**Discussion by RiteshChepuri** [PHPMailer/PHPMailer#3108] [Q&A]
How do I send Mails using php mail function. I dont want to use smtp or some other thing. I just want to send a email using native mail()
function in local host.

**Comment by Synchro** [comment]
PHPMailer uses PHP's native `mail()` functions by default. All you have to do is *not* call `isSMTP()`. See the examples. That said, sending by SMTP to localhost is safer, faster, and more reliable so you might want to re-examine why you want to do this.

**Comment by rendisimamora** [comment]
If you’re looking to send an email using PHP’s native mail() function without involving SMTP or additional libraries, here’s a straightforward example:

```
<?php
$to = "recipient@example.com";
$subject = "Subject of Your Email";
$message = "This is a test email sent from PHP's mail() function.";
$headers = "From: sender@example.com\r\n" .
           "Reply-To: sender@example.com\r\n" .
           "X-Mailer: PHP/" . phpversion();

if (mail($to, $subject, $message, $headers)) {
    echo "Email sent successfully!";
} else {
    echo "Failed to send email.";
}
?>

```
